portfolio/src/main/webapp/index.html

```diff
@@ -25,10 +25,20 @@ <h1>Tal's Portfolio</h1>
       <div id="fact-container"></div>
  
       <p>Post a comment here!</p>
+      
+      
       <div id = 'notLoggedIn'>
           <p>Login To Comment</p>
-          <button target="/user">Login</button>
+          <button target="loginUrl">Login</button>
       </div>
+      
+      
+      <div id = 'logOut'>
+          <p>Logout</p>
+          <button method="GET" action="/user" target="/" name="loggedIn" value="false">Log Out</button>
```

Comment:
This button also doesn't do anything. It should be changing the var of "loggedIn" from my /user server to false (thus logging out).